manim/utils/config_utils.py

```diff
@@ -130,23 +133,46 @@ def _parse_cli(arg_list, input=True):
         description="Animation engine for explanatory math videos",
         epilog="Made with <3 by the manim community devs",
     )
+    min_argvs = 4 if "py" in sys.argv[0] else 2
+    if input and (len(sys.argv) > min_argvs-1 # If "manim" is not the only command
+        and any(a == item for a in sys.argv for item in NON_ANIM_UTILS) #non-anim exists
+        or len(sys.argv) == min_argvs-1):
+
+        subparsers = parser.add_subparsers(dest="subcommands")
+        cfg_related = subparsers.add_parser('cfg')
+        cfg_subparsers = cfg_related.add_subparsers(dest="cfg_subcommand")
+
+        cfg_write = cfg_subparsers.add_parser('write')
+        cfg_write.add_argument(
+            "--level",
+            choices=["user", "cwd"],
+            default=None,
+            help="Specify if this config is for user or just the working directory."
+            )
+        cfg_show = cfg_subparsers.add_parser('show')
+
+        cfg_export = cfg_subparsers.add_parser("export")
+        cfg_export.add_argument("--dir",default=os.getcwd())
+
     if input:
-        parser.add_argument(
-            "file", help="path to file holding the python code for the scene",
-        )
-        parser.add_argument(
-            "scene_names",
-            nargs="*",
-            help="Name of the Scene class you want to see",
-            default=[""],
-        )
-        parser.add_argument(
-            "-o",
-            "--output_file",
-            help="Specify the name of the output file, if "
-            "it should be different from the scene class name",
-            default="",
-        )
+        # If the only command is manim, or if there are only rendering related commands
+        if len(sys.argv) < 2 or not(any(sys.argv[1] == item for item in NON_ANIM_UTILS)):
```

Comment:
Suggestion: As before, perhaps put this condition in a separate function for readability.

Reply:
In the newest commit, this isn't even necessary anymore, the function for the previous conditional takes care of it and is just `not`ed over.